### DISCUSSION TITLE
[Feature Request] Integrate DeArrow: for a wholesome use of thumbnails and titles

### DISCUSSION THREAD

**Discussion by gab12** [ReVanced/revanced-discussions#1024] [General]
DeArrow is an open-source project dedicated to crowdsourcing improved titles and thumbnails on YouTube. The objective is to ensure accuracy in titles and reduce sensationalism, eliminating elements like arrows, ridiculous faces, and clickbait.

This project is incredible and very practical. It is developed by Ajay Ramachandran, a well-known figure as the developer of SponsorBlock.

I believe integrating this functionality into Revanced would be beneficial, enhancing the overall browsing experience. An API is available and functional. DeArrow is already integrated into open-source YouTube clients such as LibreTube and Clipious.

Here is the link: [https://dearrow.ajay.app/](https://dearrow.ajay.app/)

![Capture d'écran 2024-02-05 134542](https://github.com/ReVanced/revanced-discussions/assets/6320898/5da866f2-959b-4c95-81df-388f125d34c8)

**Comment by oSumAtrIX** [comment]
A patch for alternative thumbnails exists already and also features DeArrow

**Reply by ilonachan** [reply]
But that's missing A LOT of features from DeArrow. The title for example, or the ability to see a little icon if thumbnails have been replaced and what the original one was.

**Reply by oSumAtrIX** [reply]
I disagree. The thumbnails are what makes up most

**Reply by ilonachan** [reply]
And I disagree with that, the titles are very important too. And being able to see the original thumbnail is also helpful sometimes.

**Reply by LisoUseInAIKyrios** [reply]
There already is a patch request to add titles.

**Reply by oSumAtrIX** [reply]
@ilonachan You are free to PR a titles patch.

**Reply by ilonachan** [reply]
> There already is a patch request to add titles.

That's basically all I wanted to know, thanks! It's [this one](https://github.com/ReVanced/revanced-patches/issues/2662), right?
